medusa/providers/torrent/json/t411.py

```diff
@@ -92,20 +188,38 @@ def search(self, search_strings, age=0, ep_obj=None):
 
             for search_string in search_strings[mode]:
                 if mode != 'RSS':
-                    logger.log('Search string: {search}'.format
-                               (search=search_string), logger.DEBUG)
+                    season = episode = None
+                    if isinstance(search_string, tuple):
```

Comment:
No its not.

First of all, even if it was necessary, you should check for the most general type possible, (e.g. collections.Iterable or collections.Sequence) that way it will work equally for other sequences (like a list).

Secondly, if the search api was updated to account for the new parameters (as I mentioned above) this check would be unnecessary.

Lastly, even as implemented its a better check to see that its `not isinstance(x, six.string_types)` to avoid iterating strings (whether unicode or bytestrings) and allow any other iterable.